Sources/SwiftInfo/main.swift

```diff
@@ -29,7 +53,7 @@ public struct Main {
         task.standardError = FileHandle.standardError
 
         task.terminationHandler = { t -> Void in
-            exit(t.terminationStatus)
+            SwiftInfo.exit()
```

Comment:
I think this should still be `exit(t.terminationStatus)`. This is the interruption of the subprocess, not Swiftinfo itself.

Reply:
we can use the following:
```
task.terminationHandler = { t -> Void in 
   Swiftinfo.exit(withError: t.terminationStatus == 0 ? nil : SwiftInfoError.generic("\(t.terminationStatus)"))
}
```
I wanted to use `standardError` however it's of type `any` which we don't really know if it would actually return an error confirming to the protocol `Error`

Reply:
I see. It should be fine to keep it like you did in this case.